ddtrace/__init__.py

```diff
@@ -3,7 +3,14 @@
 
 ModuleWatchdog.install()
 
-from ._logger import configure_ddtrace_logger  # noqa: E402
+# Acquire a reference to the threading module. Some parts of the library (e.g.
+# the profiler) might be enabled programmatically and therefore might end up
+# getting a reference to the tracee's threading module. By storing a reference
```

Comment:
```suggestion
# getting a reference to the tracer's threading module. By storing a reference
```

Reply:
This is indeed the tracee's module. Instead we want the tracer's module.